test/router.ts

```diff
@@ -142,9 +143,9 @@ routes:
     matchers:
       title: ["test"]
     output:
-      type: "alerts"${v}
+      type: "metrics"${v}
       dimensions: ["dim1", "dim2"]
-      stat_type: "gauge"
+      value: "hihi"
```

Comment:
discussed offline -- this is probably too slow to do in the app, and we are already robust to this in our pipeline